### PR TITLE
Add shortcut to save wiki page

### PR DESCRIPTION
**Pull request by michaelkuhn**
This uses the accesskey attribute to allow saving wiki pages using a shortcut. This functionality is typically available via Alt + accesskey.

### CODE REVIEW

**Comment by jonasfranz** [comment]
We already have a similar feature/function like this allowing you to submit a textfield form by pressing Ctrl + Enter. You just have to add `js-quick-submit` class to the the textarea. See #2540

**Comment by michaelkuhn** [comment]
Thanks for the hint! Sadly, this does not seem to work in this case (probably due to the CodeMirror stuff?). It also does not work if I add it to the internal CodeMirror textarea.

**Comment by michaelkuhn** [comment]
I have updated the PR. The `js-quick-submit` class is now added to CodeMirror's internal input field, which seems to work (my mistake before was adding it after `initCtrlEnterSubmit` had already run).

**Review by lafriks — APPROVED** [review]

**Review by jonasfranz — APPROVED** [review]

File: public/js/index.js [hunk 1]
```diff
@@ -825,6 +825,8 @@ function initWikiForm() {
                 "link", "image", "table", "horizontal-rule", "|",
                 "clean-block", "preview", "fullscreen"]
         })
+        var inputfield = simplemde.codemirror.getInputField();
+        $(inputfield).addClass("js-quick-submit");
```
Comment (lafriks) [inline public/js/index.js]: I think these two lines can be simply changed to:
```
$(simplemde.codemirror.getInputField()).addClass("js-quick-submit");
```

Comment (michaelkuhn) [reply public/js/index.js]: Good point. I have pushed an update to merge the two lines.

**Review by lafriks — APPROVED** [review]

**Pull request merged**